.github/workflows/release.yml

```diff
@@ -55,6 +41,8 @@ jobs:
           publish: yarn release
```

Comment:
this should use a fork, make sure to copy the right version and repo of the action